CHANGELOG.md

```diff
@@ -1,13 +1,22 @@
 # Changelog
 
-## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/1.27.0...HEAD)
+## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/1.28.0...HEAD)
+
+
+## [1.28.0](https://github.com/OpenLineage/OpenLineage/compare/1.27.0...1.28.0)
 
 ### Added
 
 * **Java: enable specifying custom SSL context** [`#3444`](https://github.com/OpenLineage/OpenLineage/pull/3444) [@pawel-big-lebowski](https://github.com/pawel-big-lebowski)
   *Enable providing configuration for SSL context within HTTP transport.
 * **Spark: make Spark nodes filtering configurable.** [`#3442`](https://github.com/OpenLineage/OpenLineage/pull/3442) [@pawel-big-lebowski](https://github.com/pawel-big-lebowski)
   *Spark integration filters OpenLineage events for specific plan node classes. This can be now extended with extra config entries: `allowedSparkNodes` and `deniedSparkNodes`. See [Spark Configuration documentation](https://openlineage.io/docs/integrations/spark/configuration/spark_conf) for more details. *
+* **Java: add task queue based async circuit breaker.** [`#3437`](https://github.com/OpenLineage/OpenLineage/pull/3437) [@aritrabandyo](https://github.com/aritrabandyo)
+  *This circuit breaker that executes task on a queue backed threadpool, gives up tasks if the queue is full, and keeps track of rejected tasks.*
+* **dbt: added initial support for Trino adapter** [`#3429`](https://github.com/OpenLineage/OpenLineage/pull/3429) [@whitleykeith](https://github.com/whitleykeith)
+  *This allows Trino integration to emit proper events containing Trino datasets.*
+* **Spark: increased coverage for Spark DML commands** [`#3430`](https://github.com/OpenLineage/OpenLineage/pull/3429) [@ssanthanam185](https://github.com/ssanthanam185)
```

Comment:
The link and the actual PR number don't match.

Reply:
Fixed.